client/CurrentProduct.jsx

```diff
@@ -87,7 +106,8 @@ class CurrentProduct extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <Banner>CurrentProduct
+        <Banner>
+          Current<br></br>Product
```

Comment:
Just a minor changes needed. On line 125, it looks like you are displaying the number of rating rather than the number of reviews. Please confirm :)

Reply:
Good catch!!
I put the number of ratings in there because I don't actually have any reviews in my db.
Now that I think about it, I can generate a random number for the count of reviews, and have the number of stars be relative to that number. Thanks!